### PR TITLE
Line chart styles & modals

### PR DESCRIPTION
**Pull request by noahmanger**
It's ugly, but this includes some new styles to support the [new chart work](https://github.com/18F/openFEC-web-app/pull/1508). 

- Activates nav links to "raising" and "spending" overviews, that are set to anchor links on /data
- Adds styles for line chart:
image
- Refactors `.overview__totals` as `.snapshot`, which accompanies the chart:
image
- Adds styles for modals:
image
- Note: modals require setting `main[aria-hidden="true"] { display: block }` because the modal library sets `aria-hidden="true"` on the `main` element when the modal is open, but we want it to still be visible behind the modal

### CODE REVIEW

**Comment by noahmanger** [comment]
Ready for review. Updated with better description.

File: js/templates/nav-data.hbs [hunk 1]
```diff
@@ -19,13 +19,13 @@
...
-      <div class="mega__column is-disabled">
+      <div class="mega__column">
         <div class="mega-heading">
           <h3 class="mega-heading__title icon-heading--overviews">Interactive overviews</h3>
         </div>
         <ul>
-          <li class="mega__item is-disabled">Fundraising</li>
-          <li class="mega__item is-disabled">Spending</li>
+          <li class="mega__item"><a href="{{webAPpUrl}}#raising">Raising</a></li>
+          <li class="mega__item"><a href="{{webAPpUrl}}#spending">Spending</a></li>
```
Comment (adborden) [inline js/templates/nav-data.hbs]: Typo here, `webAppUrl`

Comment (adborden) [inline js/templates/nav-data.hbs]: hmm, or should those be there at all? Maybe these should just be fragment urls?

**Comment by adborden** [comment]
This looks beautiful. Is there a partner PR that goes with this, and has @jenniferthibault taken a look?

**Comment by noahmanger** [comment]
Thanks! Yep, we paired up on it. PR over here: https://github.com/18F/openFEC-web-app/pull/1508

Change coming to those comments asap.

**Comment by noahmanger** [comment]
Ok! Fixed the `webAppUrl` name and the modal:
image

File: scss/components/_modals.scss [hunk 2]
```diff
@@ -0,0 +1,41 @@
+// Modal
+
+.modal-overlay {
```
Comment (xtine) [inline scss/components/_modals.scss]: Is this overriding a class set in the modal library or is there a reason why it isn't double dash like `.modal--overlay` to keep with convention?

Comment (noahmanger) [inline scss/components/_modals.scss]: Oh nope. That was just a mistake.

Comment (noahmanger) [inline scss/components/_modals.scss]: Fixed

**Comment by adborden** [comment]
@xtine would you mind testing the modal for the issue you spotted and merge this once you're :+1: ?

**Comment by xtine** [comment]
Modals are solid! 👍

**Pull request merged**